src/components/Design/design.module.scss

```diff
@@ -0,0 +1,20 @@
+.designContainer {
+  display: flex;
+}
+.fontBasestyle {
+  font-family: 'Dela Gothic One';
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: -0.04em;
+  line-height: 120%;
+  font-size: 32px;
+}
```

Comment:
1. should be `fontBaseStyle`
2. the default value of `font-style` is `normal` and `font-weight` is `400`, so don't need to be set